test/index.js

```diff
@@ -132,14 +139,14 @@ describe('.toV3()', function () {
   var uuid = Buffer.from('7e59dc028d42d09db29aa8a0f862cc81', 'hex')
 
   it('should work with PBKDF2', function () {
-    var key = Buffer.from('efca4cdd31923b50f4214af5d2ae10e7ac45a5019e9431cc195482d707485378', 'hex')
+    var key = Buffer.from(fixturePrivKey, 'hex')
     var wallet = Wallet.fromPrivateKey(key)
```

Comment:
Could just use `fixtureWallet` here.

Reply:
This has been done on a later commit, comment is outdated.